src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
+import { PostInfo } from '../PostInfo';
+import { Post } from '../../react-app-env';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  posts: Post[]
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <ul className="PostList">
+    {posts.map((post: Post) => (
+      <li>
+        <PostInfo post={post} />
+      </li>
```

Comment:
You should add `key` attribute to `<li>`. And also I'm not sure you need `<li>` tag at all, I think it's better to use given html structure